components/uuf-renderablecreator-hbs/src/test/java/org/wso2/carbon/uuf/handlebars/HbsHelperTest.java

```diff
@@ -119,7 +120,8 @@ public void testFaviconWithMultipleParameters() {
     @Test
     public void testTitle() {
         RequestLookup requestLookup = createRequestLookup();
-        createRenderable("{{title \"page-title\"}}").render(null, createLookup(), requestLookup, createAPI());
+        createRenderable("{{title \"page-title\"}}").render(null, createLookup(), requestLookup,
```

Comment:
Seems like this line doesn't exceed 120 limit. Hence, revert this formatting change.